src/components/admin/PortalAddEditForm.tsx

```diff
@@ -84,6 +128,8 @@ export const PortalAddEditForm: React.FC<PortalAddEditFormProps> = ({
       portal?.type,
       portal?.isEnabled,
       portal?.spaceId,
+      portal?.width_percent,
+      portal?.height_percent,
```

Comment:
[Q] Why aren't these camelCased already?
Is this PR a good opportunity or is it too much of a fix, thus OOS?

I think even if in DB it must remain backward compatible as `width_percent` and `height_percent` it is not necessarily useful to propagate the convention onto components.